### PR TITLE
feat(governance): display spot market details on proposal page

### LINKED ISSUE #4245: Support new proposal types for spot markets
## Story

As a vega user creating or managing a spot market
I want governance site to acknowledge and support the proposal types 
So I can create and update spot markets

## Acceptance Criteria

- [ ] Ensure governance app can show spot market proposals
- [ ] Ensure governance app can submit spot market proposals via the JSON submitter

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

### PR DESCRIPTION
**Pull request by asiaznik**
# Related issues 🔗

Closes #4245 

# Description ℹ️

* displays info about new spot market proposal (same as new market)
* displays info about update spot market proposal (same as update market)

# Demo 📺

Screenshot 2024-04-17 at 17 25 02
Screenshot 2024-04-17 at 17 25 10
Screenshot 2024-04-17 at 17 25 18

image

# Technical 👨‍🔧

N/A

### CODE REVIEW

**Comment by daro-maj** [comment]
https://github.com/vegaprotocol/vega/issues/11154

File: apps/governance/src/routes/proposals/Proposals.graphql [hunk 1]
```diff
@@ -383,6 +383,122 @@ fragment IUpdateNetworkParameterFields on UpdateNetworkParameter {
   }
 }
 
+fragment INewSpotMarket on NewSpotMarket {
+  instrument {
+    name
+    code
+    product {
+      ... on SpotProduct {
+        name
```
Comment (jeremyletang) [inline apps/governance/src/routes/proposals/Proposals.graphql:392]: This likely do not exist anymore in the last preview. could you test that code again staging please?

**Comment by JonRay15** [comment]
Does this cover the case where the create / update proposal is standalone and when they are part of a batch?

**Review by jeremyletang — APPROVED** [review]

**Comment by daro-maj** [comment]
> Does this cover the case where the create / update proposal is standalone and when they are part of a batch?

Yes We have checked single and batch

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**